src/components/application_manager/include/application_manager/application_manager_impl.h

```diff
@@ -1443,7 +1446,7 @@ class ApplicationManagerImpl
    * will send TTS global properties to HMI after timeout
    */
   std::map<uint32_t, TimevalStruct> tts_global_properties_app_list_;
-
+  std::set<connection_handler::DeviceHandle> QuerryApps_devices;
```

Comment:
@Ypostolov please rename according to our naming rules

Reply:
fixed inccdfc0c216991a5251ce103d0ef920c9d5dbc9c5